src/routes/townhalls/townhalls.test.ts

```diff
@@ -27,11 +27,19 @@ describe('townhall routes', () => {
             expect(status).toStrictEqual(200);
         });
     });
+    describe('/townhalls/:townhallId/bills', () => {
```

Comment:
never got the chance to see if the request could fail; all the other test suites failed

Reply:
~What do you mean by all of the other test suites failed?  I have some blank files and those technically fail since they have nothing in them, but your tests should still be running and you should be able to see the results.~

Merge with the latest master.  I noticed there's things that might make it hard to clearly see the results of your tests.  It should now be easier.